MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 recursive-include evals *.py
 recursive-include evals *.yaml
 recursive-include evals *.sql
+recursive-include evals/registry/data *.jsonl
```

Comment:
@jwang47 - this change will fetch the data when pip installing from the repo, which resolves the issue this PR was opened for without relying on dynamic loading. I will close this PR for now, then opening a separate one with just this change. Maybe we revisit dynamic loading down the line, as right now it isn't immediately needed.